packages/portal/document-viewer/src/components/nodes/Image.vue

```diff
@@ -1,14 +1,9 @@
 <template>
-  <figure>
-    <img
-      :alt="alt"
-      :src="url"
-      :title="title"
-    >
-    <figcaption>
-      <slot />
-    </figcaption>
-  </figure>
+  <img
+    :alt="alt"
+    :src="url"
+    :title="title"
+  >
```

Comment:
```suggestion
  />
```
isn't this the right way?

Reply:
<img width="804" alt="Screenshot 2023-03-02 at 7 19 01 AM" src="https://user-images.githubusercontent.com/40131297/222454174-0f105d8a-5790-412e-988a-799c41cfd838.png">

TIL

Reply:
TIL too